### PR TITLE
update car price service and controller

### PR DESCRIPTION
**Pull request by ingridarev**

### CODE REVIEW

File: project/src/main/java/com/carlease/project/car/CarServiceImpl.java [hunk 1]
```diff
@@ -24,8 +25,18 @@ public Car findById(long id) {
...
         return cars.stream().map(Car::getModel).distinct().toList();
     }
+
+    @Override
+    public Car updatePrice(long id, Car car) throws CarNotFoundException {
+        Car existingCar = carRepository.findById(id).orElseThrow(() -> new CarNotFoundException("id"));
+
+        existingCar.setPriceFrom(car.getPriceFrom());
+        existingCar.setPriceTo(car.getPriceTo());
+        return carRepository.save(existingCar);
```
Comment (grkarolis) [inline project/src/main/java/com/carlease/project/car/CarServiceImpl.java:39]: Do not expose DB entities as API contract. Bad practice. Have separate entity as API contract

File: project/src/main/java/com/carlease/project/car/ICarService.java [hunk 2]
```diff
@@ -1,10 +1,12 @@
 package com.carlease.project.car;
 import com.carlease.project.user.User;
+import com.carlease.project.user.exceptions.CarNotFoundException;
 
 import java.util.List;
 
 public interface ICarService {
```
Comment (grkarolis) [inline project/src/main/java/com/carlease/project/car/ICarService.java:7]: I notation is mostly used or found in C#. You can opt out and have just `CarService` as interface no need `I`

**Review by grkarolis — CHANGES_REQUESTED** [review]
Please do the changes or create separate tasks to do that

**Review by Teanquisitor — APPROVED** [review]

**Review by Teanquisitor — APPROVED** [review]

**Review by ryckutis — APPROVED** [review]

**Review by ryckutis — APPROVED** [review]

File: project/src/main/java/com/carlease/project/car/CarDto.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package com.carlease.project.car;
+
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+public class CarDto {
+    private long id;
```
Comment (grkarolis) [inline project/src/main/java/com/carlease/project/car/CarDto.java:9]: For later, please create a task and add documentation to API. I see that there is nothing right now. Description of the fields with example values are very appreciated

**Review by grkarolis — APPROVED** [review]

**Pull request merged**